test/cmp_ctx_test.c

```diff
@@ -137,7 +137,8 @@ static int msg_total_size_log_cb(const char *func, const char *file, int line,
                                  OSSL_CMP_severity level, const char *msg)
 {
     msg_total_size += strlen(msg);
-    TEST_note("total=%d len=%zu msg='%s'\n", msg_total_size, strlen(msg), msg);
+    TEST_note("total=%d len=%d msg='%s'\n",
+	    msg_total_size, (int)strlen(msg), msg);
```

Comment:
Hrm. Why this change?

Reply:
I don't know why I thought `%zu` wasn't standard.  Might as well undo that change and fix the tab issue at the same time.